libs/datagrid/src/lib/cells/market-name-cell.tsx

```diff
@@ -0,0 +1,40 @@
+import type { MouseEvent } from 'react';
+import { useCallback } from 'react';
+import get from 'lodash/get';
+import { Link } from 'react-router-dom';
+
+const MARKET_PATH = 'markets';
+
+interface MarketNameCellProps {
+  value?: string;
+  data?: { id?: string; marketId?: string; market?: { id: string } };
+  idPath?: string;
+  onMarketClick?: (marketId: string, metaKey?: boolean) => void;
+}
+
+export const MarketNameCell = ({
+  value,
+  data,
+  idPath,
+  onMarketClick,
+}: MarketNameCellProps) => {
+  const id = data ? get(data, idPath ?? 'id', 'all') : '';
+  const marketLink = `/${MARKET_PATH}/${id}`;
```

Comment:
chore (blocking): remove router implementation details. Lets just use the onMarketClick callback and not render a link with a url